webplugin/js/app/kommunicate-ui.js

```diff
@@ -778,6 +790,18 @@ KommunicateUI = {
         }
     },
     triggerCSAT: function () {
+        window.KM_WidgetEvents.onRateConversationClick(
+            JSON.stringify({
```

Comment:
is stringify required?

Reply:
are we returning a JSON string or an object to users when they subscribe to a specific event?

Reply:
return object

Reply:
okay